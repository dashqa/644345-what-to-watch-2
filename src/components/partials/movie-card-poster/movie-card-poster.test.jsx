import React from 'react';
import renderer from 'react-test-renderer';
import MovieCardPoster from './movie-card-poster';

const props = {
  name: ``,
  posterImage: ``,
  isBig: false,
};

it(`Movie card poster correctly renders after relaunch`, () => {
  const tree = renderer.create(<MovieCardPoster {...props}/>).toJSON();
  expect(tree).toMatchSnapshot();
});
