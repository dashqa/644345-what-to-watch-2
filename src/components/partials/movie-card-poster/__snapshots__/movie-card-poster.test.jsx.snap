// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Movie card poster correctly renders after relaunch 1`] = `
<div
  className="movie-card__poster"
>
  <img
    alt=""
    height="327"
    src=""
    width="218"
  />
</div>
`;
